.circleci/config.yml

```diff
@@ -1,39 +1,131 @@
 version: 2.1
+references:
+  postgres: &postgres
+    image: circleci/postgres:9.5-alpine
+    environment:
+      POSTGRES_USER: circleci-demo-ruby
+      POSTGRES_DB: rails_blog_test
+      POSTGRES_PASSWORD: ""
 
 orbs:
-  cli: circleci/circleci-cli@0.1.4
+  ruby: circleci/ruby@<<pipeline.parameters.dev-orb-version>>
+  orb-tools: circleci/orb-tools@9.0
 
-workflows:
-  main:
-    jobs:
-      - test:
-          filters:
-            tags:
-              # Simplified SemVer regex
-              only: /^\d+\.\d+\.\d+$/
-      - publish:
-          filters:
-            branches:
-              ignore: /.*/
-            tags:
-              # Simplified SemVer regex
-              only: /^\d+\.\d+\.\d+$/
-          context: orb-publishing
+# Pipeline parameters
+parameters:
+  # These pipeline parameters are required by the "trigger-integration-tests-workflow"
+  # job, by default.
+  run-integration-tests:
+    type: boolean
+    default: false
+  dev-orb-version:
+    type: string
+    default: "dev:alpha"
 
 jobs:
-  test:
-    executor: cli/default
+  integration-tests:
+    docker:
+      - image: cimg/ruby:2.7-node
+      - *postgres
+    parallelism: 2
```

Comment:
This is not something we need to address in this PR, but I don't think this is working right. In the last pipeline for this branch, one container ran 1 test while the other ran 30 tests.

Reply:
Timing data issue. I should hopefully have just corrected it.

Reply:
Actually the timings are split evenly. It looks as if maybe a single test is very long.